includes/Modules/Analytics.php

```diff
@@ -1206,7 +1207,7 @@ private function print_tracking_opt_out() {
 			<!-- <?php esc_html_e( 'End Google Analytics AMP opt-out snippet added by Site Kit', 'google-site-kit' ); ?> -->
 		<?php else : ?>
 			<!-- <?php esc_html_e( 'Google Analytics opt-out snippet added by Site Kit', 'google-site-kit' ); ?> -->
-			<script type="text/javascript">window["ga-disable-<?php echo esc_attr( $property_id ); ?>"] = true; </script>
+			<?php BC_Functions::wp_print_inline_script_tag( sprintf( 'window["ga-disable-%s"] = true;', esc_attr( $property_id ) ) ); ?>
```

Comment:
Try this locally first as I think PHPCS will complain about something but, the same idea from my last comment.
```suggestion
			<?php BC_Functions::wp_print_inline_script_tag(
				sprintf( 'window["ga-disable-%s"] = true;', esc_attr( $property_id ) )
			); ?>
```

Reply:
@aaemnnosttv Ended up being this after running phpcbf-

```php
                        <?php
			BC_Functions::wp_print_inline_script_tag(
				sprintf( 'window["ga-disable-%s"] = true;', esc_attr( $property_id ) )
			);
			?>
```

Acceptable?